pkg/config/config_template.yaml

```diff
@@ -1,857 +1,1238 @@
+  ## potentially sensitive information.
+  ## Each rules has to contain:
+  ##  * name - string - The tag name to replace, for services or resources use "service.name" or "resource.name".
+  ##  * pattern - string - The pattern to match the desired content to replace
+  ##  * repl - string - what to inline if the pattern is matched
+  ##
+  ## For instance to remove all query paramters from the http.url tag you would use:
+  ##
+  ##   - name: "http.url"
+  ##     pattern: "\?.*$"
+  ##     repl: "?"
+  ##
+  #
+  # replace_tags:
+  #   - name: "<TAG_NAME>"
+  #     pattern: "<REGEX_PATTERN>"
+  #     repl: "<PATTERN_TO_INLINE>"
+
+  ## @param ignore_resources - list of strings - optional
+  ## A blacklist of regular expressions can be provided to disable certain traces based on their resource name
+  ## all entries must be surrounded by double quotes and separated by commas.
+  #
+  # ignore_resources: ["(GET|POST) /healthcheck"]
+
+{{ end -}}
+{{- if .ProcessAgent }}
+
+######################################
+## Process Collection Configuration ##
+######################################
+
+## @param process_config - custom object - optional
+## Enter specific configurations for your Process data collection.
+## Uncomment this parameter and the one below to enable them.
+## See https://docs.datadoghq.com/graphing/infrastructure/process/
 #
+# process_config:
+
+  ## @param enabled - string - optional - default: "false"
+  ##  A string indicating the enabled state of the Process Agent:
+  ##    * "false"    : The Agent collects only containers information.
+  ##    * "true"     : The Agent collects containers and processes information.
+  ##    * "disabled" : The Agent process collection is disabled.
+  #
+  # enabled: "true"
+
+  ## @param expvar_port - string - optional - default: 6062
+  ## Port for the debug endpoints for the process-agent.
+  #
+  # expvar_port: 6062
+
+  ## @param log_file - string - optional
+  ## The full path to the file where process-agent logs are written.
+  #
+  # log_file: <PROCESS_LOG_FILE_PATH>
+
+  ## @param intervals - custom object - optional - default: 10s for normal checks and 2s for others.
+  ## The interval, in seconds, at which the Agent runs each check. If you want consistent
+  ## behavior between real-time, set the `container_realtime` and `process_realtime` intervals to 10.
+  #
+  # intervals:
+  #   container: 10
+  #   container_realtime: 2
+  #   process: 10
+  #   process_realtime: 2
+
+  ## @param blacklist_patterns - list of strings - optional
+  ## A list of regex patterns that exclude processes if matched.
+  #
+  # blacklist_patterns:
+  #   - <REGEX>
+
+  ## @param queue_size - integer - optional - default: 20
+  ## How many check results to buffer in memory when POST fails.
+  #
+  # queue_size: 20
+
+  ## @param max_per_message - integer - optional - default: 100
+  ## The maximum number of processes or containers per message.
+  #
+  # max_per_message: 100
```

Comment:
Unused?

Reply:
ping @olivielpeau i'm removing this parameter since indeed it seems to be not used.

Reply:
it's used in another repo

Reply:
Sorry, I didn't know about `datadog-process-agent`! Newbie mistake 😃

Reply:
Now, reviewing [yaml_config.go from datadog-process-agent](https://github.com/DataDog/datadog-process-agent/blob/master/config/yaml_config.go), the following entries are missing:
```
process.strip_proc_arguments
process_config.windows.args_refresh_interval
process_config.windows.add_new_args
process.additional_endpoints
process.container_source
process.intervals.connections
process.expvar_port
```